index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>PTI Docs</title>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify-darklight-theme@latest/dist/style.min.css" title="docsify-darklight-theme" type="text/css" />
+</head>
+<body>
+<p>
+    <a href="api/v0/index.html">API reference</a>
```

Comment:
Did you put the latest version from pti-api openapi spec ?

Reply:
yes, I generated it